presto-release/src/main/java/com/facebook/presto/release/git/GitCommandsBinder.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.release.git;
+
+import com.facebook.presto.release.AbstractAnnotatedProvider;
+import com.google.inject.Binder;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+
+import java.lang.annotation.Annotation;
+
+import static com.facebook.airlift.configuration.ConfigBinder.configBinder;
+import static java.util.Objects.requireNonNull;
+
+public class GitCommandsBinder
```

Comment:
For people who don't know how dependency injection works this is hard to understand without looking at external resources.

A short explanation of what the binding step is would be helpful.

Reply:
Added the following comments to the class
```
   // To perform the binding: gitBinder(binder).bind(annotation, repositoryName)
```